README.md

```diff
@@ -47,17 +47,52 @@ This proposal is a collection of multiple packages.
 
 ### Goals [or Motivating Use Cases, or Scenarios]
 
-[What is the end-user need which this project aims to address?]
+- Bring some of the benefits that Wasm provides to CPU code, over to GPU code. 
```

Comment:
Might want to clarify this - we're not explicitly addressing GPU ISAs which this statement could imply

Reply:
Fair point, feel free to edit the text